### DISCUSSION TITLE
Energy distribution view needlessly complex

### DISCUSSION THREAD

**Discussion by i-am-shodan** [home-assistant/frontend#20139] [Dashboard]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [X] I have tried reproducing the issue in [safe mode](https://www.home-assistant.io/blog/2023/11/01/release-202311/#restarting-into-safe-mode) to rule out problems with unsupported custom resources.

### Describe the issue you are experiencing

To my mind the energy distribution graph doesn’t accurately represent power distribution when off peak charging a battery at night and discharging this during the day. The colours are a bit confusing too.

To fix this I’d like the option of massively simplifying how the data is presented by just using solar production and grid usage. This would give just two colours of ~50% each. Alternatively the ability to override the colours so I can mark the battery as solar (although I suspect this will result in an inaccurate graph)

As an example here is my power usage for today. 

![IMG_3724](https://github.com/home-assistant/frontend/assets/6901273/c5978203-8d9e-41e4-848e-70aff297bb85)

The colours here are horrible, 3 shades of greeny blue. This is a great view but what I really want to quickly see is how much solar I’ve used as a %.

### Describe the behavior you expected

Simplify (or the option to) just use energy production and energy from grid when rendering the home bubble.

### Steps to reproduce the issue

1.
2.
3.
...

### What version of Home Assistant Core has the issue?

2024.3.0

### In which browser are you experiencing the issue with?

All

### Which operating system are you using to run this browser?

All

**Comment by karwosts** [comment]
This is not a bug, and more appropriate for the feature requests/discussions.
